tests/behat/features/bootstrap/EventContext.php

```diff
@@ -541,6 +541,31 @@ public function createAnonymousEventEnrollees(TableNode $eventAnonymousEnrollees
     }
   }
 
+  /**
+   * Enable the calendar button with a given calendar.
+   *
+   * @Given ":calendar" calendar is enabled
+   */
+  public function iDisableVerifiedImmediately(string $calendar) {
```

Comment:
@nechai I guess we need to rename this method ;)

Reply:
Done 😅